django_socio_grpc/management/commands/generateproto.py

```diff
@@ -119,8 +126,20 @@ def handle(self, *args, **options):
                 if self.generate_pb2:
                     if not settings.BASE_DIR:
                         raise ProtobufGenerationException(detail="No BASE_DIR in settings")
-                    os.system(
-                        f"python -m grpc_tools.protoc --proto_path={settings.BASE_DIR} --python_out=./ --grpc_python_out=./ {file_path}"
+                    extra_args = options["extra_args"] if options["extra_args"] else ""
+                    # INFO - AM - 16/04/2024 - subprocess.run is safe because we are not using shell=True. Please do not change this. Unit test check this
+                    subprocess.run(
```

Comment:
We could use the module directly https://github.com/grpc/grpc/blob/master/tools/distrib/python/grpcio_tools/grpc_tools/protoc.py#L209